content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md

```diff
@@ -0,0 +1,167 @@
+---
+reviewers:
+- michmike
+- patricklang
+title: Adding Windows nodes
+min-kubernetes-server-version: 1.17
+content_template: templates/tutorial
+weight: 30
+---
+
+{{% capture overview %}}
+
+The Kubernetes platform can now be used to run both Linux and Windows containers. This page shows how one or more Windows nodes can be registered to a cluster.
```

Comment:
This change would be a _nice-to-have_.
I'd like to make it clear that the whole Pod has to be a single platform (sorry, no Windows Server sidecar for your Linux container just yet).
```suggestion
You can use Kubernetes to run a mixture of Linux and Windows nodes, so you can mix Pods that run on Linux on with Pods that run on Windows. This page shows how to register Windows nodes to your cluster.
```